documentation/modules/virt-migration-workflow.adoc

```diff
@@ -30,10 +32,30 @@ The workflow describes the following steps:
 +
 *After the VM disks are transferred:*
 
-. The `Migration Controller` service creates a `conversion` pod with the PVCs attached to it.
+. The `Migration Controller` service creates a `conversion` pod with the PVCs attached to it when importing from VMWare.
 +
 The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the target VM.
++
+. The `Migration Controller` service creates a `VirtualMachine` CR for each source virtual machine (VM), connected to the PVCs.
+
+. When the target VM is powered on, the `KubeVirt Controller` service creates a `virt-launcher` pod and a `VirtualMachineInstance` CR.
++
+The `virt-launcher` pod runs `QEMU-KVM` with the PVCs attached as VM disks.
 
+*Cold migration:*
```

Comment:
This is relevant for ovirt/openstack. But, in VMware it's different now (also the above flow). Shall I add it?

Reply:
@ahadas - please can we ask for your opinion?

Reply:
yes please. no need to get too much into the details - just to indicate that in that specific flow the conversion pod also imports the disks rather than CDI

Reply:
updated, @nyoxi please confirm.